docker/templates/default.conf.template

```diff
@@ -15,9 +15,9 @@ server {
         index  index.html index.htm;
         try_files $uri /index.html;
 
-        # application has inline styles
         # use INLINE_RUNTIME_CHECK=false yarn build and nonce / hash to apply a CSP
-        #add_header Content-Security-Policy "default-src 'none'; script-src 'self'; connect-src 'self'; img-src 'self'; font-src fonts.gstatic.com; style-src 'self' fonts.googleapis.com;" always;
+        # Use add_header Content-Security-Policy "......." always;
+        add_header Content-Security-Policy-Report-Only "default-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval'; style-src 'self' fonts.googleapis.com 'unsafe-inline'; img-src 'self' data:; font-src 'self' fonts.gstatic.com" always;
```

Comment:
If all is well after this is deployed, we will circle back and take off the "-Report-Only" part of the header key.